Samples/Server/Unity-FractalStream/Assets/Scripts/FlipY.cs

```diff
@@ -0,0 +1,11 @@
+﻿using UnityEngine;
+
+public class FlipY : MonoBehaviour
```

Comment:
can you xmldoc this so folks can understand what it can be used for when they look at our sample

Reply:
Sure, my bad.